tests/utils/backend-utils.ts

```diff
@@ -1,64 +1,43 @@
-import { execSync, spawn, SpawnOptionsWithoutStdio } from 'child_process';
-import { sleep } from './index.js';
+import {LocalProtocolSubstrate, UsageMode} from "@webb-tools/test-utils";
+import isCi from "is-ci";
+import path from "path";
+import {ApiPromise} from "@polkadot/api";
+
+const usageMode: UsageMode = isCi
+  ? { mode: 'docker', forcePullImage: false }
+  : {
+    mode: 'host',
+    nodePath: path.resolve(
+      '../../protocol-substrate/target/release/webb-standalone-node'
+    ),
+  };
 
-function spawnWithLogger(command: string, options?: SpawnOptionsWithoutStdio, allLogs = false) {
-  const process = spawn(command, options);
+let aliceNode: LocalProtocolSubstrate;
```

Comment:
Can we return the nodes in startWebbNodes(), and the user can retrieve an api from them, and stop them in the tests? It seems weird to have global variables in the utils folder.